.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
+profile=black
+src_paths=snuba
+multi_line_output=3
+line_length=88
```

Comment:
Why 88?

Reply:
That's black's limit as well. Otherwise the two will conflict.